blueocean-dashboard/src/main/js/components/Pipelines.jsx

```diff
@@ -68,6 +91,10 @@ export class Pipelines extends Component {
                                 { organization && orgLink }
                             </h1>
                         </Extensions.Renderer>
+                        
+                        {fastSearchFeatureFlag &&
+                            <TextInput className="search-pipelines-input" iconLeft="search" defaultValue={this.getSearchText()} placeholder="Search pipelines..." onChange={this.onChange} />   
```

Comment:
Based on the discussion in jenkinsci/jenkins-design-language#170 I don't think we can get the correct fix for `TextInput` in a fast timeframe and I wouldn't want to block this feature because of that. You may want to change this back to a raw HTML `input` that uses the styles for `TextInput` but binds the `value` attribute manually.